lib/twilio.rb

```diff
@@ -0,0 +1,11 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+auth_token = ENV['TWILIO_AUTH_TOKEN']
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+@client.account.messages.create(
+body: "Thank you! Your order was placed and will be delivered before #{(Time.now + (60 * 60)).asctime}",
```

Comment:
Line is too long. [104/80]<br>Indent the first parameter one step more than the previous line.